central_v2/src/components/EGMostPopular.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+import { Grid, Typography, Box, styled } from '@mui/material';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import { APIClients, IGameTemplate } from '@righton/networking';
+import { v4 as uuidv4 } from 'uuid'; 
+import StyledGameCard from './GameCard';
+import { ScreenSize } from '../lib/HostModels';
+import placeHolder from '../images/placeHolder.svg';
+
+
+interface EGMostPopularProps {
+  screenSize: ScreenSize;
+  apiClients: APIClients;
+  searchedGames: IGameTemplate[];
+}
+
+interface MostPopularTextProps {
+    screenSize: ScreenSize;
+  }
+
+const MostPopularText = styled(Typography)(({ screenSize }: MostPopularTextProps) => ({
+  lineHeight: screenSize === ScreenSize.SMALL ? '36px' : '60px',
+  fontFamily: 'Poppins',
+  fontWeight: '700',
+  fontSize: screenSize === ScreenSize.SMALL ? '24px' : '40px',
```

Comment:
This `fontSize` follows the Figma right? I ask because we don't normally see font size changes across device sizes.